meta-mender-core/recipes-mender/mender-client/mender-client_3.5.2.bb

```diff
@@ -1,4 +1,4 @@
-require mender-client.inc
+require mender-client-go.inc
```

Comment:
This commit you can skip by just dropping https://github.com/mendersoftware/meta-mender/pull/2045/commits/3ba8e26f38d6280911c140595627195261ebf29f from the cherry-pick list.